api/v1alpha1/mirrorpeer_types.go

```diff
@@ -52,19 +50,11 @@ type MirrorPeerSpec struct {
 	// +kubebuilder:validation:MaxItems=2
 	// +kubebuilder:validation:MinItems=2
 	Items []PeerRef `json:"items"`
-	//  The default MirroringMode is snapshot which is configured on the rbd images.
-	// +kubebuilder:validation:Enum=snapshot;journal
-	// +kubebuilder:validation:Optional
-	// +kubebuilder:default=snapshot
-	Mode MirroringMode `json:"mirroringMode,omitempty"`
-	// The ReplicationSecretName contains the ceph user credentials to connect to ceph cluster for mirroring operations
-	// +kubebuilder:validation:Optional
-	// +kubebuilder:default="rook-csi-rbd-provisioner"
-	ReplicationSecretName string `json:"replicationSecretName,omitempty"`
+
 	// The SchedulingInterval in which the mirroring snapshot are taken
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:default="5m"
-	SchedulingInterval string `json:"schedulingInterval,omitempty"`
+	SchedulingInterval []string `json:"schedulingInterval,omitempty"`
```

Comment:
```suggestion
	SchedulingIntervals []string `json:"schedulingInterval,omitempty"`
```

Reply:
CRD doesn't reflect this change. Update it.

Reply:
updated